client/src/helpers/nova/hooks/useAnchorAddressState.ts

```diff
@@ -56,8 +62,15 @@ export const useAnchorAddressState = (address: AnchorAddress): [IAnchorAddressSt
         initialState,
     );
 
-    const { anchorOutput, isLoading: isAnchorDetailsLoading } = useAnchorDetails(network, address.anchorId);
-    const { totalBalance, availableBalance } = useAddressBalance(network, state.addressDetails, anchorOutput);
+    const { anchorOutput, anchorOutputMetadata, isLoading: isAnchorDetailsLoading } = useAnchorDetails(network, address.anchorId);
+    const { manaRewards } = useOutputManaRewards(network, anchorOutputMetadata?.outputId ?? "");
```

Comment:
We can remove `manaRewards`  for anchor output because the `getRewards` returns the totally available Mana rewards of an account or delegation output.